src/Microsoft.TestPlatform.CrossPlatEngine/EventHandlers/TestRequestHandler.cs

```diff
@@ -24,6 +24,8 @@ namespace Microsoft.VisualStudio.TestPlatform.CommunicationUtilities
     public class TestRequestHandler : ITestRequestHandler
     {
         private int protocolVersion = 1;
+        
+        // Also check TestRequestSender.
```

Comment:
Nit: We probably need to clarify this comment. It took a while for me to understand it.

```suggestion
        // Must be in sync with the highest supported version in src/Microsoft.TestPlatform.CommunicationUtilities/TestRequestSender.cs file.
```

Reply:
Done.